includes/User.php

```diff
@@ -2685,6 +2685,14 @@ public function isAnon() {
 	}
 
 	/**
+	 * Whether this user is Wikia staff or not
+	 * @return bool
+	 */
+	public function isStaff() {
+		return in_array( 'staff', $this->getEffectiveGroups() );
```

Comment:
It may be time to dedicate a constant to staff, say, `GROUP_STAFF`? Imagine not having to grep for `"staff"/'staff'` anymore!